### PR TITLE
restaurant menu: mics UI improvements

### PR DESCRIPTION
**Pull request by vladimir-8**
- fixed: plus button display
- fixed: make more menu scrollable
- fixed: image align center only on mobile
- fixed: allow button content to wrap
- updated: product details layout
- added: cart title on mobile
- display badges and tags separately

### CODE REVIEW

File: js/app/restaurant/components/Order/MobileOrderHeading.js [hunk 1]
```diff
@@ -39,7 +42,7 @@ export default function MobileOrderHeading() {
         }) }
         onClick={ handleClick }>
         <div className="panel-heading__body">
-          { isMobileCartVisible ? null : (<OrderState />) }
+          { isMobileCartVisible ? (<span>{t('CART_TITLE')}</span>) : (<OrderState />) }
```
Comment (Atala) [inline js/app/restaurant/components/Order/MobileOrderHeading.js:45]: it is what I see when I have the mobile cart closed, is it expected? (display warning about order total) 

image

Comment (Atala) [inline js/app/restaurant/components/Order/MobileOrderHeading.js:45]: Mobile cart title is yellow like a warning it is expected ?

image

**Review by Atala — APPROVED** [review]

File: js/app/restaurant/components/Order/MobileOrderHeading.js (hunk 1 above)
Comment (vladimir-8) [reply js/app/restaurant/components/Order/MobileOrderHeading.js:45]: Yes, it comes from the `<OrderState />` component

Comment (vladimir-8) [reply js/app/restaurant/components/Order/MobileOrderHeading.js:45]: Yes, it's the same `<OrderState />` component 😄

**Pull request merged**